### PR TITLE
ci: publish suspicious cases nuget

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+        echo RELEASE_VERSION=$(cat semver) >> $GITHUB_ENV
+      shell: bash
+  
+  build-api-suspicious-cases:
+    name: Build Api Suspicious Cases
+    uses: Informatievlaanderen/build-pipeline/.github/workflows/build-image.yml@main
+    needs: [ set-release-version ]
+    if: ${{ (github.repository_owner == 'Informatievlaanderen') && (needs.set-release-version.outputs.version != 'none') }}
+    with:
+      registry: integration-db
```
Comment (jvandaal) [inline .github/workflows/release.yml:62]: Verify registry

File: build.fsx [hunk 2]
```diff
@@ -43,85 +41,24 @@ supportedRuntimeIdentifiers <- [ "msil"; "linux-x64" ]
...
-  let dist = (buildDir @@ "AddressRegistry.CacheWarmer" @@ "linux")
-  let source = "src" @@ "AddressRegistry.CacheWarmer"
-
-  Directory.ensure dist
-  Shell.copyFile dist (source @@ "Dockerfile")
-  containerize "AddressRegistry.CacheWarmer" "cache-warmer")
+    "Basisregisters.IntegrationDb.SuspiciousCases.Api"
+  ] |> List.iter publishSource)
+
+Target.create "Containerize_ApiIntegrationDb" (fun _ -> containerize "Basisregisters.IntegrationDb.SuspiciousCases.Api" "api-suspicious-cases")
```
Comment (jvandaal) [inline build.fsx:61]: Should be the same name as in the release.yml Containerize_ApiSuspiciousCases

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+      build-target: Containerize_ApiSuspiciousCases
```
Comment (jvandaal) [inline .github/workflows/release.yml:63]: Change to what is in the build.fsx

File: build.fsx [hunk 4]
```diff
@@ -25,7 +23,7 @@ let product = "Basisregisters Vlaanderen"
 let copyright = "Copyright (c) Vlaamse overheid"
 let company = "Vlaamse overheid"
 
-let dockerRepository = "address-registry"
+let dockerRepository = "integration-db"
```
Comment (jvandaal) [inline build.fsx:26]: Verify

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,577 @@
+name: Release
```
Comment (jvandaal) [inline .github/workflows/release.yml:1]: Commit message should start with "ci". For example ci: publish suspicious cases nuget

File: .github/workflows/release.yml (hunk 1 above)
Comment (ArneD) [reply .github/workflows/release.yml:62]: don't build yet?

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,577 @@
...
+    needs: [ set-release-version ]
+    if: ${{ (github.repository_owner == 'Informatievlaanderen') && (needs.set-release-version.outputs.version != 'none') }}
+    with:
+      pack-file: Be.Vlaanderen.Basisregisters.IntegrationDb.SuspiciousCases.Api
+      test-project: Basisregisters.IntegrationDb.SuspiciousCases.Api.Tests
+      build-project: Basisregisters.IntegrationDb.SuspiciousCases.Api
+      semver: ${{ needs.set-release-version.outputs.version }}
+    secrets: inherit
+
+  # release:
```
Comment (ArneD) [inline .github/workflows/release.yml:83]: you will need to release to publish

**Review by ArneD — APPROVED** [review]

**Review by jvandaal — APPROVED** [review]

**Pull request merged**